linux-tools/kpartx/kpartx.sh

```diff
@@ -86,7 +86,8 @@ function run_kpartx_tests()
 
     tc_register "Test kpartx -d"
     kpartx -d $kpartx_img >$stdout 2>$stderr
-    grep "loop deleted" $stdout | grep -q  /dev/$loopdev
+    # wait for max 10 sec to update
+    tc_wait_for_file_text $stdout "loop deleted : /dev/$loopdev"
     tc_fail_if_bad $? "kpartx -d fail1" || return
```

Comment:
If you can add the failure string that comes in stderr , it gives a better understanding why the change is required.
Few things:

- Can we add a check for "kpartx -d" first ?
kpartx -d $kpartx_img >$stdout 2>$stderr
tc_fail_if_bad $? "kpartx -d failed" || return 

- Followed by verifying the expected message in stdout 
tc_wait_for_file_text $stdout "loop deleted : /dev/$loopdev"

Also correct the error mesage: "kpartx -d fail1"